india_banking/india_banking/doctype/bank_connector/bank_connector.py

```
@@ -121,7 +127,11 @@ def make_post_request(self, payment_order, otp=None, action=None):
 
 			payload = self.get_payload(payment_order, otp=otp)
 
-			response = request.post(url, headers=headers, data=json.dumps(payload))
+			response = None
+			if self.same_site == "Yes":
+				response = self.handle_same_site_request(payload)
+			else:
+				response = request.post(url, headers=headers, data=json.dumps(payload))
```

Ruff (0.15.1)

[error] Probable use of `requests` call without timeout

(S113)